src/gui/org/deidentifier/arx/gui/resources/Resources.java

```diff
@@ -45,8 +45,11 @@
 
 /**
  * This class provides access to basic resources.
+ *  
+ * Images by FAMFAMFAM, http://www.famfamfam.com/.

```

Comment:
Why? None of the icons that existed prior to this PR are from FAMFAMFAM (whatever this is) and I don't think that you needed to use this website to draw a black circle around them, right?

Reply:
That is unrelated to the four new icons. I thought all icons are from this source.